pkg/cloudprovider/providers/openstack/openstack_routes.go

```diff
@@ -146,7 +147,10 @@ func (r *Routes) CreateRoute(ctx context.Context, clusterName string, nameHint s
 
 	onFailure := newCaller()
 
-	addr, err := getAddressByName(r.compute, route.TargetNode)
+	IP, _, _ := net.ParseCIDR(route.DestinationCIDR)
+	CIDRisV6 := IP.To4() == nil
```

Comment:
Both `IP` and `CIDRisV6` should have (at least) a lowercased first letter, to conform with golang's public/private styleguide/rules.